raiden/ui/cli.py

```diff
@@ -167,6 +166,7 @@ def check_synced(blockchain_service):
         tolerance=ORACLE_BLOCKNUMBER_DRIFT_TOLERANCE,
         sleep=3,
     )
+    return net_id
```

Comment:
do we need to return the id here? I tried to make the functions starting with `check_` consistent by calling `sys.exit()`, would be good to avoid returning anything but `None`